src/components/Card/CardMedia.tsx

```diff
@@ -50,7 +50,7 @@ function CardMedia<El extends HTMLElement = HTMLDivElement>({
     <CardMediaWrapper
       $isImageComponent={isImageComponent}
       $isMediaComponent={isMediaComponent}
-      alt={as === 'img' && hasMediaSrc ? alt : undefined}
+      alt={alt}
```

Comment:
I think that the condition should stay there as the `alt` attribute doesn't exist on tags other than `img`.  This way the `alt` attribute will be always there when the property is defined it's defined